packages/frontapp/src/app/components/UserCard/index.spec.tsx

```diff
@@ -40,7 +40,7 @@ describe('UserCard Component', () => {
   });
 
   it('should render loading', async () => {
-    expect(screen.getByText('Loading')).toBeInTheDocument();
+    expect(screen.getByTestId('oval-loading')).toBeInTheDocument();
```

Comment:
Guys, I would recommend using a role selector to check for this condition
`screen.getByRole('status')`

Read [this](https://testing-library.com/docs/queries/about/#priority)

Reply:
For the next time, @vyacheslav1618 , please use approach suggested by @ngame2011 . For now, I don't see any major problems with this. Let's merge and go ahead.